src/servers/src/http/influxdb.rs

```diff
@@ -46,30 +46,40 @@ pub async fn influxdb_write(
     Query(mut params): Query<HashMap<String, String>>,
     lines: String,
 ) -> Result<impl IntoResponse> {
-    let db = params
```

Comment:
I think we can refactor this function to accept the `db` parameter and add two new functions `influxdb_write_v1` and `influxdb_write_v2` that extract the `db` value from different params. 
Under high writing load, this logic to extract `db` value is an unnecessary cost.